src/bindings/python/wheel/setup.py

```diff
@@ -381,6 +382,11 @@ def generate_package(self, src_dirs):
 class CopyExt(build_ext):
     """Copy extension files to the build directory."""
 
+    user_options = [
+        ("skip-rpath", None, "Skips RPATH for Python extensions."),
+    ]
+    boolean_options = ["skip_rpath"]
+    
     def run(self):
         if len(self.extensions) == 1:
```

Comment:
please, revert all changes in this file.

Reply:
Done.